cmd/armadactl/cmd/docs.go

```diff
@@ -43,10 +48,45 @@ func docsCmd() *cobra.Command {
 				return fmt.Errorf("could not render content from documentation file: %s", err)
 			}
 
-			fmt.Println(out)
+			if term.IsTerminal(int(os.Stdout.Fd())) {
+				if err = openTextViewer(out); err != nil {
+					return fmt.Errorf("could open documentation in text viewer: %s", err)
```

Comment:
- Change `could` to `could not`
- If it can't open the text viewer suggest
   - Write that message as a warning to stderr
   - Dump the output to stdout